### PR TITLE
Add proxy instructions

### PR DESCRIPTION
**Pull request by yahavi**
https://github.com/yahavi/jfrog-vscode-extension/tree/proxy-instructions#proxy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
 Search in tree:
 ![Search_In_Tree](resources/readme/gifs/search.gif)
 
-To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button.
+To filter scan results, click on the Filter ![Filter](resources/readme/filter.png) button:
 ![Filter](resources/readme/gifs/filter.gif)
 
+### Proxy
```
Comment (eyalbe4) [inline README.md]: ### Proxy configuration

File: README.md [hunk 2]
```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
...
+
+To use HTTP/S proxy:
```
Comment (eyalbe4) [inline README.md]: To use HTTP/S proxy: --> If your JFrog Xray instance is behind an HTTP/S proxy, follow these steps to configure the proxy server.

File: README.md [hunk 3]
```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
...
+
+1. Go to Preferences --> Settings --> Application --> Proxy
+1. Set the proxy URL under 'Proxy'.
+1. Make sure 'Proxy Support' is 'override' or 'on'.
+
+- As alternative, you can use the http_proxy and https_proxy environment variables.
```
Comment (eyalbe4) [inline README.md]: As alternative --> Alternatively

Shouldn't the env vars be upper case? (HTTP_PROXY and HTTPS_PROXY).

File: README.md [hunk 4]
```diff
@@ -39,9 +39,41 @@ View dependency in package.json:
...
+
+#### Proxy Authorization
+
+To use authenticated proxy:
```
Comment (eyalbe4) [inline README.md]: To use authenticated proxy: --> If your proxy servers requires credentials, follow these steps.

File: README.md (hunk 3 above)
Comment (yahavi) [reply README.md]: Its case insensitive. Changing to uppercase to be more clear.

**Pull request merged**